app/viewmodels/SummaryCardHelper.scala

```diff
@@ -43,13 +43,14 @@ class SummaryCardHelper @Inject()() extends ImplicitDateFormatter with ViewUtils
     penalties.map { penalty =>
       val newPenalty = findAndReindexPointIfIsActive(indexedActivePoints, penalty)
       (newPenalty.penaltyCategory, newPenalty.appealInformation.flatMap(_.head.appealStatus)) match {
-        case (LSPPenaltyCategoryEnum.Point, _) if newPenalty.FAPIndicator.contains("X") && newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Active => addedPointCard(newPenalty, thresholdMet) //Added FAP
-        case (LSPPenaltyCategoryEnum.Point, _) if newPenalty.FAPIndicator.contains("X") && newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive => removedPointCard(newPenalty) //Removed FAP
-        case (LSPPenaltyCategoryEnum.Point, Some(AppealStatusEnum.Upheld)) if newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive => pointSummaryCard(newPenalty, thresholdMet) //Appealed point
-        case (LSPPenaltyCategoryEnum.Point, _) if newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive && newPenalty.expiryReason.isDefined => removedPointCard(newPenalty) //Removed point
-        case (LSPPenaltyCategoryEnum.Point, _) => pointSummaryCard(newPenalty, thresholdMet) // normal points
-        case (LSPPenaltyCategoryEnum.Threshold, _) => financialSummaryCard(newPenalty, threshold) //normal threshold
-        case (LSPPenaltyCategoryEnum.Charge, _) => financialSummaryCard(newPenalty, threshold)
+          //TODO: More robust handling of missing information
+        case (Some(LSPPenaltyCategoryEnum.Point) | None, _) if newPenalty.FAPIndicator.contains("X") && newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Active => addedPointCard(newPenalty, thresholdMet) //Added FAP
+        case (Some(LSPPenaltyCategoryEnum.Point) | None, _) if newPenalty.FAPIndicator.contains("X") && newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive => removedPointCard(newPenalty) //Removed FAP
+        case (Some(LSPPenaltyCategoryEnum.Point) | None, Some(AppealStatusEnum.Upheld)) if newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive => pointSummaryCard(newPenalty, thresholdMet) //Appealed point
+        case (Some(LSPPenaltyCategoryEnum.Point) | None, _) if newPenalty.penaltyStatus == LSPPenaltyStatusEnum.Inactive && newPenalty.expiryReason.isDefined => removedPointCard(newPenalty) //Removed point
+        case (Some(LSPPenaltyCategoryEnum.Point) | None, _) => pointSummaryCard(newPenalty, thresholdMet) // normal points
```

Comment:
Is the `| None` needed here?

Reply:
Andrew and I discussed how we should handle a None for the category and treating it as a Point, since we can differentiate between scenarios based on other data. I've left a TODO to make it more robust, which I think we'll need to address in a future ticket

Reply:
I would have thought that 'normal' points would have a penalty category?